### PR TITLE
init VoiceOfFaust at v0.7

### PR DESCRIPTION
**Pull request by magnetophon**

### CODE REVIEW

File: pkgs/applications/audio/VoiceOfFaust/default.nix [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    cp PAFvocoder $out/bin/
+    cp stringSinger $out/bin/
+    cp subSinger $out/bin/
+    #cp VocSynthFull $out/bin/
+    mkdir $out/PureData/
+    cp PureData/OscSendVoc.pd $out/PureData/OscSendVoc.pd
+  '';
+
+  meta = {
+    description = "turn your voice into a synthesizer";
```
Comment (pSub) [inline pkgs/applications/audio/VoiceOfFaust/default.nix]: Descriptions should start with a captial letter.

File: pkgs/applications/audio/VoiceOfFaust/default.nix [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  src = fetchFromGitHub {
+    owner = "magnetophon";
+    repo = "VoiceOfFaust";
+    rev = "${version}";
+    sha256 = "14jjs7cnhg20pzijgblr7caspcpx8p8lpkbvjzc656s9lqn6m9sn";
+  };
+
+  plugins = [ helmholtz mrpeach ];
+
+  pitchTracker = callPackage ../puredata/wrapper.nix { inherit plugins; };
```
Comment (pSub) [inline pkgs/applications/audio/VoiceOfFaust/default.nix]: Not sure if it makes sense to add pitchTracker to all-packages.nix

File: pkgs/applications/audio/VoiceOfFaust/default.nix [hunk 3]
```diff
@@ -0,0 +1,63 @@
+{ stdenv, pkgs, callPackage, fetchFromGitHub, faust2jack, helmholtz, mrpeach }:
+stdenv.mkDerivation rec {
+  name = "VoiceOfFaust-${version}";
+  version = "v0.7";
```
Comment (pSub) [inline pkgs/applications/audio/VoiceOfFaust/default.nix]: Better move the "v" to the rev attribute of `fetchFromGitHub`, as versions should start with a digit. See http://nixos.org/nixpkgs/manual/#idm140737319248160

File: pkgs/applications/audio/VoiceOfFaust/default.nix (hunk 2 above)
Comment (magnetophon) [inline pkgs/applications/audio/VoiceOfFaust/default.nix]: @pSub Are you saying I should add it to all-packages.nix?  Or do you mean this somehow adds it?

**Pull request merged**